### PR TITLE
Fix Storybook

### PR DESCRIPTION
**Pull request by 8845musign**
# Overview

- Error when starting Storybook and opening page
  - `Failed to fetch dynamically imported module: http://localhost:6006/src/stories/Accordion.stories.tsx`
- It worked by removing react-swc
  - I don't understand the cause.
- And while we're at it, we also upgraded Storybook.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "@ubie/design-tokens": "0.1.4",
     "@ubie/prettier-config": "0.1.0",
     "@ubie/ubie-icons": "^0.5.0",
-    "@vitejs/plugin-react-swc": "^3.6.0",
+    "@vitejs/plugin-react": "^4.2.1",
```
Comment (8845musign) [inline package.json:73]: Main modification

File: .storybook/preview.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import '@ubie/design-tokens/dist/tokens.css';
 
 export const parameters = {
-  actions: { argTypesRegex: '^on[A-Z].*' },
```
Comment (8845musign) [inline .storybook/preview.js:4]: migration for v8

**Comment by kodai3** [comment]
@8845musign 

So, in my environment, it still not reproduce. Just works fine. 🤔
```
// delete storybook cache
rm -rf node_modules/.cache/storybook
// run the storybook
npm run story book
```
You might want to try the cache deleting if you haven't. 

I found the issue looks similar to your error. https://github.com/storybookjs/storybook/issues/21610

maybe this workaround will solve for now.
```diff
// vite.config.ts
export default defineConfig({
  plugins: [react()],
+  resolve: {
+    // default plus mdx ref: https://github.com/storybookjs/storybook/issues/21610#issuecomment-1882417258
+    extensions: ['.mjs', '.js', '.mts', '.ts', '.jsx', '.tsx', '.json', '.mdx'],
+  },
});
```

**Comment by 8845musign** [comment]
@kodai3 

> rm -rf node_modules/.cache/storybook

I fixed it by re-creating node_modules!
I guess `npm ci` alone didn't work.

Thank you!!!!!

**Pull request closed without merging**